tuna/miopen/miopen_lib.py

```diff
@@ -57,7 +58,9 @@ def parse_args(self):
     parser = setup_arg_parser(
         'Run Performance Tuning on a certain architecture', [
             TunaArgs.ARCH, TunaArgs.NUM_CU, TunaArgs.VERSION,
-            TunaArgs.CONFIG_TYPE, TunaArgs.SESSION_ID
+            TunaArgs.CONFIG_TYPE, TunaArgs.SESSION_ID, TunaArgs.MACHINES,
+            TunaArgs.REMOTE_MACHINE, TunaArgs.LABEL, TunaArgs.RESTART_MACHINE,
+            TunaArgs.DOCKER_NAME
```

Comment:
Would it make sense to move the ownership or arguments to MIOpen? Also, is `DOCKER_NAME` used anywhere ?

Reply:
yeah, i dont really like how arguments are split between all scripts/miopen/example/go_fish but i dont want to address this here. Ill try and do some improvements in a future PR where i introduce some yaml parsing

Reply:
Lets revisit this when you get to it